LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 GetInData
```

Comment:
Actually these brackets make sense if you look at the section above ;-)

Reply:
right :)